.github/workflows/test.yml

```diff
@@ -79,7 +79,7 @@ jobs:
           helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
             --namespace=default \
             --set=licenseKey=${{ secrets.NEW_RELIC_LICENSE_KEY }} \
-            --set=controllerManager.manager.image.tag=${{ inputs.K8S_OPERATOR_IMAGE_TAG }}
+            --set=controllerManager.manager.image.version=${{ inputs.K8S_OPERATOR_IMAGE_TAG }}
```

Comment:
Is this and the Makefile changes an artifact of Hannah's PR #144? Should they be included in this PR?

Reply:
They're required for the new operator version to work so I pulled it in.